app/src/main/java/org/simple/clinic/bloodsugar/unitselection/BloodSugarUnitSelectionUiActions.kt

```diff
@@ -1,5 +1,8 @@
 package org.simple.clinic.bloodsugar.unitselection
 
+import org.simple.clinic.bloodsugar.BloodSugarUnitPreference
+
 interface BloodSugarUnitSelectionUiActions {
   fun closeDialog()
+  fun prefilledBloodSugarUnitSelection(blodSugarUnitPreference: BloodSugarUnitPreference)
```

Comment:
The name of this method is in the past tense, that is it implies that _something has happened_ instead of telling the UI action of an operation that must be performed. Can we rename this method to better reflect this, please?